src/client.ts

```diff
@@ -193,4 +199,13 @@ export class AutoblocksAPIClient {
   }): Promise<DatasetWithItems> {
     return this.get(`/datasets/${args.datasetId}`);
   }
+
+  public async getTestCases(args: {
```

Comment:
can you add an e2e test to https://github.com/autoblocksai/javascript-sdk/blob/main/e2e/test-api-client-esm/index.js?

the e2e tests still need to be moved to `test/e2e` but you can just add it there for now if you want